Mod Menu Helper/assets/modmenu/lang/tr_tr.json

```diff
@@ -5,40 +5,45 @@
   "modmenu.summaryTranslation.colormatic": "OptiFine'ın özel renkleri\n§9✎§8 Gökyüzü/yer boşluğunu temizle, ışık yumuşatma",
-  "modmenu.summaryTranslation.continuity": "OptiFine'ın bağlantılı dokuları\n§9✎§8 Modu, metodu, kaynak paketlerini aç/kapa",
+  "modmenu.summaryTranslation.continuity": "OptiFine'ın bağlantılı dokuları\n§9✎§8 Modu, yöntemi, kaynak paketlerini aç/kapa",
   "modmenu.summaryTranslation.cullleaves": "OptiFine'ın \"daha iyi ağaçları\"\n§9✎§8 Modu, kaynak paketini aç/kapa",
   "modmenu.summaryTranslation.cem": "OptiFine'ın özel varlıkları (alfa)\n§9✎§8 OptiFine desteği, model düzeltimi",
   "modmenu.summaryTranslation.dynamicfps": "Arkaplandayken FPS'i düşür\n§9✎§8 Modu aç/kapa, üzerindeyken normale çevir, ...",
-  "modmenu.summaryTranslation.enhancedblockentities": "Daha hızlı sandık, tabela, yataklar vb.\n§9✎§8 Geliştirilecek bloklar, metod",
+  "modmenu.summaryTranslation.enhancedblockentities": "Daha hızlı sandık, tabela, yataklar vb.\n§9✎§8 Geliştirilecek bloklar, yöntem",
   "modmenu.summaryTranslation.entity_texture_features": "OptiFine'daki rastgele ve ışık emen moblar\n§9✎§8 Rastgele moblar, ışık emiciliği, göz kırpabilmeleri, ...",
-  "modmenu.summaryTranslation.entityculling": "Engellerin arkasındaki mobları görüntüleme\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.entityculling": "Duvarların arkasındaki mobları önizlemeyi durdur\n§8✎ (sadece gelişmiş seçenekler)",
   "modmenu.summaryTranslation.fabrishot": "F9'la yüksek çözünürlüklü görüntü al\n§9✎§8 F2, çözünürlük, tuşlar vb. üzerine yazma",
   "modmenu.summaryTranslation.farsight": "Sunuculardaki yüklü yığınları yüklü tutar\n§8✎ (sadece gelişmiş seçenekler)",
   "modmenu.summaryTranslation.fastchest": "§c*§r Daha hızlı sandıklar\n§9✎§8 Modu aç/kapa",
   "modmenu.summaryTranslation.ferritecore": "RAM kullanımı azaltır\n§8✎ (sadece gelişmiş seçenekler)",
   "modmenu.summaryTranslation.iris": "OptiFine'ın shaderları\n§9✎§8 Modu aç/kapa, shader, tuş atamaları vb. seç",
-  "modmenu.summaryTranslation.itemmodelfix": "Elde tutulan eşyadaki görünmez çizgileri düzeltir\n§9✎§8 Metod",
+  "modmenu.summaryTranslation.itemmodelfix": "Elde tutulan eşyadaki görünmez çizgileri düzeltir\n§9✎§8 Yöntem",
   "modmenu.summaryTranslation.lambdabettergrass": "OptiFine'daki \"daha iyi çimenler\"\n§9✎§8 Kalite, kar, kaynak paketi",
   "modmenu.summaryTranslation.lambdynlights": "OptiFine'ın \"dinamik ışıklandırmaları\"\n§9✎§8 Kalite, varlık seçenekleri, ...",
   "modmenu.summaryTranslation.languagereload": "Hızlı dil değiştirme, arama çubuğu\n§9✎§8 Dili yeniden yüklemek için F3+J",
   "modmenu.summaryTranslation.lazydfu": "Daha hızlı oyun açılışı\n§8✎ (seçenekleri yok)",
   "modmenu.summaryTranslation.lithium": "Fizikleri ve yığınları optimize eder\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.isxander-main-menu-credits": "Modpack sürümü başlık ekranında\n§8✎ (sadece gelişmiş seçenekler)",
```

Comment:
I'm not really sure that we should keep "modpack" word as it is or translate into "mod paketi" or "modpaketi"?

Reply:
Updated "modpack" word